.hound.yml

```diff
@@ -0,0 +1,26 @@
+# Example .hound.yml for a Node.js, Express, Postgres, and React on our e-commerce project
+
+
+eslint:
+  enabled: true
+  config_file: .eslintrc.json
```

Comment:
Where is this file? Please create it and then configure for hound to use ES6 features, because hound is commenting on them as violations.

Reply:
Already the file .hound.yml